tictactoe.py

```diff
@@ -31,17 +32,21 @@
 	"bg.png": "https://raw.githubusercontent.com/Tashar02/tic-tac-toe/main/bg.png",
 }
 
-for filename, url in assets.items():
+# Downloads a file if it does not exist
+def download_asset(filename, url):
```

Comment:
Since there are multiple assets, naming it download_assets sounds more appropriate imo.